src/components/constants.ts

```diff
@@ -148,64 +148,82 @@ export const TIME_UNIT_OPTIONS = [
 export const TEN_MINUTES_IN_MS = 1000 * 60 * 10;
 export const FIVE_MINUTES_IN_MS = 1000 * 60 * 5;
 
-const EXAMPLE_SCRIPT_SCRIPTED = btoa(`import { check, fail } from 'k6'
+const EXAMPLE_SCRIPT_SCRIPTED = btoa(`import { expect } from 'https://jslib.k6.io/k6-testing/0.5.0/index.js';
+import { check } from 'k6'
 import http from 'k6/http'
+import secrets from 'k6/secrets';
 
-export default function main() {
-  const result = http.get('http://test.k6.io/');
-
-  // console.log will be represented as logs in Loki
-  console.log('got a response');
-  
-  // Use check() to test conditions. These show as 'assertions' in the dashboard
-  // Note: failed check() calls do not impact uptime and reachability
-  const pass = check(result, {
-    'is status 200': (r) => r.status === 200,
-  });
-
-  // Use fail() to abort and fail a test, impacting uptime and reachability
-  if(!pass){
-    fail(\`non 200 result \${result.status}\`);
+// This example authenticates against the quick pizza API
+export default async function main() {
+  const authURL = 'https://quickpizza.grafana.com/api/users/token/login';
+
+  // TIP: Secure your credentials using secrets.get()
+  // https://grafana.com/docs/grafana-cloud/testing/synthetic-monitoring/create-checks/manage-secrets/
+  const authInfo = {
+    username: 'default', // username: await secrets.get('quickpizza-username');
+    password: '12345678' // password: await secrets.get('quickpizza-password');
   }
+
+  const resp = http.post( authURL, JSON.stringify(authInfo), { headers: { 'Content-Type': 'application/json' }});
```

Comment:
IMO: For readability, I'd move the headers to a `const`

```suggestion
  const headers = {
    headers: { 'Content-Type': 'application/json' },
  };

  const resp = http.post(authURL, JSON.stringify(authInfo), headers);
```

Reply:
Updated - thank you!